chatter/src/components/Messages/components/Message.js

```diff
@@ -1,23 +1,23 @@
 import React from 'react';
 import cx from 'classnames';
+import { MY_USER_ID } from '../../UserList/constants/users';
 
-const ME = 'me';
-
-export default function Message({ nextMessage, message, botTyping }) {
+const Message = React.memo(({ message, isLast }) => {
   return (
     <p
       className={cx(
         'messages__message',
         'animate__animated animate__rubberBand',
         {
-          'messages__message--me': message.user === ME,
-          'messages__message--last': (!nextMessage && (!botTyping || message.user === ME))
-            || (nextMessage && nextMessage.user !== message.user)
+          'messages__message--me': message.user === MY_USER_ID,
+          'messages__message--last': isLast
```

Comment:
Before, passing `botTyping` in will cause all `Message`s being re-rendered when bot is typing, which isn't necessary. After change, only the last one `Message` is re-rendered.